specify/specify7
# Define separate config for bulk carry forward

Fixes #5114

### Checklist

      and self-explanatory (or properly documented)

### Testing instructions

- Create a new CO record
- Save
- Click on the gear icon to access the form meta
- Enable the "Show Carry Forward Button" and "Show bulk carry forward count"
- Click on the gear icon next to "Show bulk carry forward count"
- [ ] Verify you cannot Uncheck a required field (set as required in Schema Config)
- Define you bulk carry forward config (might have to reload the page for the carry config to be applied)
- Input a value higher than 2 for "Bulk Carry Forward Count"
- Click Carry Forward 
- [ ] Verify it respected your config
